openquake/commonlib/readinput.py

```diff
@@ -656,6 +660,8 @@ def get_exposure(oqparam):
         set(['occupants'])
     asset_refs = set()
     time_event = getattr(oqparam, 'time_event', None)
+    ignore_missing_costs = set(getattr(oqparam, 'ignore_missing_costs', []))
```

Comment:
Even if the name is already long, you might consider renaming `ignore_missing_costs` -> `ignore_missing_cost_types`